### PR TITLE
Silkebonnen/ele 3345 implement new basic card

### PR DESCRIPTION
**Pull request by silkeholmebonnen**
## Issue

[ELE-3345](https://linear.app/electricitymaps/issue/ELE-3345/implement-new-basic-card)

## Description

The goal of this PR is to create a new card for estimations.

### Preview

**Estimated Dark**
image
**Estimated Dark collapsed**
image
**Estimated Light**
image
**Estimated Light collapsed**
image
**Data outage light**
image
**Data outage dark**
image
**Data aggregated and estimated dark**
image
**Data aggregated and estimated light**
image
**Data aggregated dark**
image
**Data aggregated light**
image

To replicate this set `isEstimated`, and/or `isAggregated` and/or `isOutage` to true in ZoneDetails.tsx.

### CODE REVIEW

**Comment by silkeholmebonnen** [comment]
@madsnedergaard Do you know why it is failing?

**Comment by silkeholmebonnen** [comment]
@madsnedergaard do you know why prettier is failing?

**Comment by silkeholmebonnen** [comment]
> > @madsnedergaard do you know why prettier is failing?
> 
> It don't like the parenthesis around the ternary operators, I can push the formatting fix if you want?
> 
> but then again Eslint don't like the prettier code 😅

Yes that would be great if you could do that! Thanks Viktor

File: web/src/components/Badge.tsx [hunk 1]
```diff
@@ -7,17 +7,17 @@ type BadgeProps = {
...
-    warning: 'bg-yellow-400 dark:bg-yellow-500',
+    default: 'bg-[#E5E5E5] dark:bg-[#374151]',
+    warning: 'bg-[#B45309]/20 dark:bg-[#F59E0B]/20',
   }[type];
   const textColorClasses = {
-    default: 'text-gray-800/90 dark:text-gray-200/90',
-    warning: 'text-yellow-900/95 dark:text-gray-900/95',
+    default: 'text-black dark:text-white',
+    warning: 'text-[#B45309] dark:text-[#F59E0B]',
   }[type];
```
[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/src/components/Badge.tsx]: No thanks, we use Tailwind 😅

File: web/.eslintrc.json [hunk 2]
```diff
@@ -142,7 +142,7 @@
         "ignore": [".*[pP]rops"]
       }
     ],
-    "unicorn/no-nested-ternary": ["error"],
+    "unicorn/no-nested-ternary": "off", // TODO: Change back to error when we upgrade prettier to v3+
```
[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/.eslintrc.json:145]: Argh, it was not supposed to be sent yes - sorry, will work on something!

File: web/src/components/Badge.tsx [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
-    default: 'bg-gray-300 dark:bg-gray-500/40',
-    warning: 'bg-yellow-400 dark:bg-yellow-500',
-  }[type];
-  const textColorClasses = {
-    default: 'text-gray-800/90 dark:text-gray-200/90',
-    warning: 'text-yellow-900/95 dark:text-gray-900/95',
-  }[type];
+  const bgColorClasses =
+    type == 'estimated'
+      ? 'bg-neutral-200 dark:bg-gray-700'
```
[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/src/components/Badge.tsx:12]: Also, I see that @Alportan did use the `neutral` colors in Figma, so I subtract my comment for now (and then we can discuss it during the design system update later 😄)

File: web/.eslintrc.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (VIKTORVAV99) [reply web/.eslintrc.json:145]: This was my call, I'm not really sure why we have these rules but either way they are in conflict with the prettier version we use, which we really should not introduce. Apparently parentheses in a nested ternary is fine for ESLint but not prettier and vice versa.

**Comment by q--** [comment]
Related: https://github.com/electricitymaps/electricitymaps-contrib/issues/6035

File: web/src/features/panels/zone/EstimationCard.tsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+              >
+                {GetTitle(estimationData)}
+              </h2>
+            </div>
+            <div className="flex flex-row gap-2 ">
+              {(estimationData.isEstimated || estimationData.isOutage) && (
+                <Badge type={pillType}>{GetPillText(estimationData)}</Badge>
+              )}
+              <div className="text-lg">
+                {isCollapsed ? <HiChevronUp /> : <HiChevronDown />}
```
Comment (VIKTORVAV99) [inline web/src/features/panels/zone/EstimationCard.tsx:96]: Are these not inverted?
image
image

I think I would prefer them the other way around.

**Review by madsnedergaard — COMMENTED** [review]
Looks really nice on the app and overall this is great! 🙏 

I have added some comments, some are a bit larger and requires more changes, so we can also do it together on Friday if you want? :)

File: web/src/features/panels/zone/EstimationCard.tsx (hunk 4 above)
Comment (silkeholmebonnen) [reply web/src/features/panels/zone/EstimationCard.tsx:96]: I'll ask @Alportan about it

Comment (madsnedergaard) [reply web/src/features/panels/zone/EstimationCard.tsx:113]: We couldn't figure out what's the right way here, but [Material UI](https://mui.com/material-ui/react-accordion/) and [shadcn/ui](https://ui.shadcn.com/docs/components/accordion) does as you suggest, so let's go with that 😄

File: web/.eslintrc.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/.eslintrc.json:145]: That's also weird - but I'd rather have it crash and force you to avoid doing the nested ternaries at all than this :)

**Review by madsnedergaard — APPROVED** [review]

File: web/src/features/panels/zone/EstimationCard.tsx (hunk 4 above)
Comment (Alportan) [reply web/src/features/panels/zone/EstimationCard.tsx]: Ohh good catch Viktor, I'll update Figma to match reality. 🙌

**Pull request merged**